ext/standard/basic_functions.c

```diff
@@ -1129,20 +1129,58 @@ PHP_FUNCTION(flush)
 /* }}} */
 
 /* {{{ Delay for a given number of seconds */
-PHP_FUNCTION(sleep)
-{
-	zend_long num;
-
-	ZEND_PARSE_PARAMETERS_START(1, 1)
-		Z_PARAM_LONG(num)
-	ZEND_PARSE_PARAMETERS_END();
-
-	if (num < 0) {
-		zend_argument_value_error(1, "must be greater than or equal to 0");
-		RETURN_THROWS();
-	}
-
-	RETURN_LONG(php_sleep((unsigned int)num));
+PHP_FUNCTION(sleep) {
+  zval *num;
```

Comment:
please use tabs instead of spaces, since this is the standard in C files.

Reply:
ideally there should be a clang-format conf file or something to automate this 🤔

Reply:
hmm, seems the formatting in php-src is a mess, for example compare
https://github.com/php/php-src/blob/09415077f933afe93b0645aca100eb5a0bd51812/ext/standard/array.c#L71-L72
with 
https://github.com/php/php-src/blob/09415077f933afe93b0645aca100eb5a0bd51812/ext/standard/array.c#L95
- on line 71 it's
```c
void f()
{
```
and on line 95 its
```c
void f() {
```
🤔

Reply:
Actually seems @nikic was working on fixing this back in 2020 but never finished it: https://github.com/php/php-src/commit/4fd63185803aad42790c0ac38c144b346e283009

Reply:
https://github.com/php/php-src/blob/PHP-8.3.3/.github/actions/verify-generated-files/action.yml#L15 command can be used in CI to assert if syntax is the expected after any CS fixer is applied

Reply:
formatted the code, 
and made a script to format PR's: https://github.com/php/php-src/pull/13414

Reply:
made a CI/automatic version: https://github.com/php/php-src/pull/13417